py3status/modules/mpd_status.py

```diff
@@ -106,7 +106,7 @@ class Py3status:
 
     # available configuration parameters
     cache_timeout = 2
-    format = "{state} [[[{artist}] - {title}]|[{file}]]"
+    format = "{state} [[[{artist} ]- {title}]|[{file}]]"
```

Comment:
please modify it also in the main docstring, this breaks CI

Reply:
Done! Sorry for that.